replication/binlogsyncer.go

```diff
@@ -573,13 +573,17 @@ func (b *BinlogSyncer) writeBinlogDumpMariadbGTIDCommand(gset mysql.GTIDSet) err
 	return b.writeBinlogDumpCommand(mysql.Position{Name: "", Pos: 0})
 }
 
-// localHostname returns the hostname that register slave would register as.
+// localHostname returns the hostname that register replica would register as.
+// this gets truncated to 255 bytes.
```

Comment:
because in the old version the length limit is 60, it's better to truncate based on MySQL version